### ISSUE TITLE
`collapse-footer`: Add an optional opacity-fade animation

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6832]
### Why this would be helpful

The current animation of the expanding footer that is part of the Sticky Footer addon cannot be disabled and is a motion animation. This may trigger certain users. It's also jarring when it expands when you aren't expecting.

### My suggestion

A solution for this issue would be to add an opacity-fade option, which does not involve motion.

**Comment by Secret-chest** [comment]
A fade animation is uselessly long. There should be an instant open option

**Comment by DNin01** [comment]
> A fade animation is uselessly long. There should be an instant open option

I'd make the fade animation faster than the glide-in one.

I'll have to try out no transition - at first thought, it doesn't seem quite right for such a big element, but why not have the option, as long as it's not the default? Scratch has very few motion transitions.

**Comment by Samq64** [comment]
Maybe an instant option could be added after #6833?
